src/components/DataVisualisationGuide/index.js

```diff
@@ -24,7 +24,7 @@ function DataVisualisationGuide({ title, items, ...props }) {
         <RichTypography component="h4" variant="h4" className={classes.title}>
           {title}
         </RichTypography>
-        <Hidden smDown implementation="css">
+        <Hidden mdDown implementation="css">
```

Comment:
There are other places such as these ones where we're changing breakpoints @kelvinkipruto . You still haven't stated why.